src/test/java/studytestsuiteold/Study_004_AddItemsToCartTestCase.java

```diff
@@ -0,0 +1,43 @@
+package studytestsuiteold;
+
+import org.testng.annotations.Test;
+import pageobjectsold.CartPage;
+import pageobjectsold.IndexPage;
+import utility.Constants;
+
+import static org.openqa.selenium.support.PageFactory.initElements;
+import static org.testng.Assert.assertTrue;
+import static utility.Services.ManageUrlService.getURL;
+import static utility.Services.WaiterService.waitForElementVisible;
+
+/**
+ * Created by user on 31.05.17.
+ */
+public class Study_004_AddItemsToCartTestCase extends DefaultTestCase {
+
+    @Test
+    public void test_004(){
```

Comment:
It is a test how we create tests at my work, one class with name - only one test method. At new tests I did as you wish :)